extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -255,7 +258,9 @@ class QueueProcessor extends EventEmitter {
             });
             this._consumer.on('metrics', data => {
                 // i.e. data = { my-site: { ops: 1, bytes: 124 } }
-                this._mProducer.publishMetrics(data,
+                const filteredData = {};
```

Comment:
Is there a need for this object? E.g. why not just send `data[this.site]` to `publichMetrics`?

Reply:
I think he intends to send an object containing `data[this.site]` as a key, hence the new object.

Reply:
yup, i'm trying to keep the data in the form of `{ site-name: { ... } }`

Reply:
Oh I see, it's an object that contains only the relevant key, hence it is filtered. Thanks!